views/optin.inc.php

```diff
@@ -40,6 +40,36 @@
     <tbody>\n";
 
 $bugzilla_locale_name = urlencode(Bugzilla::getBugzillaLocaleField($current_locale, 'www'));
+$bugzilla_link = function ($current_filename) use ($bugzilla_locale_name, $current_locale) {
+    if (is_array($current_filename)) {
+        // Filing a bug for multiple files
+        $bug_title = "[l10n: {$current_locale}] Add opt-in pages to '{$current_locale}'";
+        $bug_body = "Please add '{$current_locale}' to the list of supported locales for the following files on mozilla.org:\n";
+        foreach ($current_filename as $filename) {
+            $bug_body .= "* {$filename}\n";
+        }
+    } else {
+        // Filing a bug for one file
+        $bug_title = "[l10n: {$current_locale}] Add '{$current_locale}' to the list of supported locales for {$current_filename}";
+        $bug_body = "Please add '{$current_locale}' to the list of supported locales for {$current_filename} on mozilla.org";
+    }
+
+    $bug_link = 'https://bugzilla.mozilla.org/enter_bug.cgi?alias=&assigned_to=francesco.lodolo%40gmail.com' .
+                '&blocked=&bug_file_loc=http%3A%2F%2F&bug_severity=normal&bug_status=NEW' .
+                '&comment=' . urlencode($bug_body) .
+                '&component=L10N&contenttypeentry=&contenttypemethod=autodetect' .
+                '&contenttypeselection=text%2Fplain&data=&dependson=&description=&flag_type-4=X' .
+                '&flag_type-418=X&flag_type-419=X&flag_type-506=X&flag_type-507=X&form_name=enter_bug' .
+                '&keywords=&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=All' .
+                '&priority=--&product=www.mozilla.org&qa_contact=pascalc%40gmail.com' .
+                '&rep_platform=All&short_desc=' . urlencode($bug_title) .
+                '&target_milestone=---&version=Development%2FStaging' .
+                '&format=__default__&cf_locale=' . $bugzilla_locale_name;
+
+    return $bug_link;
```

Comment:
Since we also create bugzilla links in views/listsitesforlocale.inc.php do you think it would make sense to make a proper function ?